### PR TITLE
Hybrid and Pairwise Broker Feature

### PR DESCRIPTION
**Pull request by pkanher617**
This PR merges the functionality of the Hybrid flow using the spa_code and the pairwise broker flow, allowing the user to bootstrap a pairwise broker using a hybrid spa_code from a client side application. This will also introduce the pairwise broker and hybrid flow to the dev branch.

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -1 +1,12 @@
...
+always-auth=true
+
+; begin auth token
+//identitydivision.pkgs.visualstudio.com/Engineering/_packaging/MSAL-JS/npm/registry/:username=IdentityDivision
+//identitydivision.pkgs.visualstudio.com/Engineering/_packaging/MSAL-JS/npm/registry/:_password=[BASE64_ENCODED_PERSONAL_ACCESS_TOKEN]
+//identitydivision.pkgs.visualstudio.com/Engineering/_packaging/MSAL-JS/npm/registry/:email=npm requires email to be set but doesn't use the value
+//identitydivision.pkgs.visualstudio.com/Engineering/_packaging/MSAL-JS/npm/:username=IdentityDivision
+//identitydivision.pkgs.visualstudio.com/Engineering/_packaging/MSAL-JS/npm/:_password=[BASE64_ENCODED_PERSONAL_ACCESS_TOKEN]
+//identitydivision.pkgs.visualstudio.com/Engineering/_packaging/MSAL-JS/npm/:email=npm requires email to be set but doesn't use the value
+; end auth token
```
Comment (jasonnutter) [inline .npmrc]: Did you mean to include this? Think it can be discarded.

Comment (pkanher617) [reply .npmrc]: removed

**Comment by tnorling** [comment]
@pkanher617 @jasonnutter Is this just merging #3978 and #3645 into a single PR? Anything in particular we should take a closer look at here?

**Comment by jasonnutter** [comment]
> @pkanher617 @jasonnutter Is this just merging #3978 and #3645 into a single PR? Anything in particular we should take a closer look at here?

Yes...I was thinking the hybrid pr would be merged first, and then this diff will just include the broker changes.

**Comment by ghost** [comment]
Reminder: The next release is scheduled for next week and this PR appears to be stale :( 

If changes have been requested please address feedback. 
If this PR is still a work in progress please mark as draft.

**Pull request closed without merging**